UI/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+
+	<head>
+		<meta charset="UTF-8"/>
+		<title>Моя перва HTML страница</title>
+		<link href = "style.css" rel = "stylesheet" type = "text/css"/>
+	</head>
+	<body>
+		<div class = "Wallpaper"></div>
+		
+		
+		<div class = "Header"></div>
+		<div class = "HeaderIcon">HUKUMKA</div>
+		<div class = "HeaderUserName">username</div>
+		<div class = "HeaderRectangle"></div>
+		<a href = "https://vk.com/justakel" class = "HeaderLogin">sign out</a>
+		
+		
+		<div class = "Footer"></div>
```

Comment:
I see your problems with structure of HTML.

The idea of your HTML markup is to create a hierarchy of elements.
E.g. footer: let's imagine you have single footer block which contains left and right parts, each of them contains some paragraphs. In this case your markup will look like:

```
<div class="footer">
    <div class="footer-right-part">
          <p>My name</p>
          <p>my.email@mail.com</p>
    </div>
    <div class="footer-left-part">
           <p>15.03.1021</p>
           <p>Group 8</p>
    </div>
</div>
````
Also it's better to have consistency between look of the page and HTML structure: if some element is supposed to be on the very bottom of the page, please place it in the end of body tag.

In this case it will be much easier for you to style and maintain your HTML-markup.